app/javascript/components/api.js

```diff
@@ -16,3 +16,23 @@ export async function fetchProfile (id) {
     return {}
   }
 }
+
+export async function createProfile (profileData) {
+  try {
+    const options = {
+      method:'POST',
+      headers:{
+        'Content-Type':'application/json'
+      },
+      body:JSON.stringify(profileData)
+    }
+    const response = await fetch(`profiles`, options)
+      // return response.json()
```

Comment:
This response is in fact the new profile... we prolly dont need it here so i think we can just return true or something and remove the const assignment... so just put the `await fetch...` and under `return true`